arcgis-ios-sdk-samples/Content Display Logic/Controllers/Progress View Controller/ProgressViewController.xib

```diff
@@ -0,0 +1,76 @@
+                        <subviews>
+                            <stackView opaque="NO" contentMode="scaleToFill" alignment="center" translatesAutoresizingMaskIntoConstraints="NO" id="R2o-b6-Ljz">
+                                <rect key="frame" x="16" y="16" width="206.5" height="95.5"/>
+                                <subviews>
+                                    <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="16" translatesAutoresizingMaskIntoConstraints="NO" id="1K4-en-qgj">
+                                        <rect key="frame" x="0.0" y="0.0" width="206.5" height="95.5"/>
+                                        <subviews>
+                                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" horizontalCompressionResistancePriority="1000" text="Presenting status of long-running process" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="4" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="XQR-fN-MRY">
+                                                <rect key="frame" x="0.0" y="0.0" width="206.5" height="42.5"/>
+                                                <fontDescription key="fontDescription" style="UICTFontTextStyleHeadline"/>
+                                                <nil key="textColor"/>
+                                                <nil key="highlightedColor"/>
+                                            </label>
+                                            <activityIndicatorView opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" hidesWhenStopped="YES" animating="YES" style="large" translatesAutoresizingMaskIntoConstraints="NO" id="fuJ-0p-PJT">
+                                                <rect key="frame" x="0.0" y="58.5" width="206.5" height="37"/>
+                                            </activityIndicatorView>
+                                        </subviews>
+                                    </stackView>
+                                </subviews>
+                            </stackView>
+                        </subviews>
+                        <constraints>
+                            <constraint firstAttribute="trailing" secondItem="R2o-b6-Ljz" secondAttribute="trailing" constant="16" id="PwF-za-IED"/>
+                            <constraint firstAttribute="bottom" secondItem="R2o-b6-Ljz" secondAttribute="bottom" constant="16" id="Qsb-Vg-aCq"/>
+                            <constraint firstItem="R2o-b6-Ljz" firstAttribute="top" secondItem="tiG-wt-R88" secondAttribute="top" constant="16" id="Y2d-Q6-BI6"/>
+                            <constraint firstItem="R2o-b6-Ljz" firstAttribute="leading" secondItem="tiG-wt-R88" secondAttribute="leading" constant="16" id="z8T-qc-pYE"/>
```

Comment:
Instead of constraining to the superview with a constant, usually the subview is constrained to the layout margins and then the margins are specified as desired.

Reply:
I've made some changes. Please let me know if they are correct!